src/class-browser-shots-gutenberg.php

```diff
@@ -119,54 +119,58 @@ public function register_block() {
 			'browser-shots/browser-shots',
 			array(
 				'attributes'      => array(
-					'html'       => array(
+					'html'         => array(
 						'type'    => 'string',
 						'default' => '',
 					),
-					'url'        => array(
+					'url'          => array(
 						'type'    => 'string',
 						'default' => '',
 					),
-					'width'      => array(
+					'width'        => array(
 						'type'    => 'int',
 						'default' => 600,
 					),
-					'height'     => array(
+					'height'       => array(
 						'type'    => 'int',
 						'default' => 450,
 					),
-					'alt'        => array(
+					'alt'          => array(
 						'type'    => 'string',
 						'default' => '',
 					),
-					'link'       => array(
+					'link'         => array(
 						'type'    => 'string',
 						'default' => '',
 					),
-					'target'     => array(
+					'target'       => array(
 						'type'    => 'string',
 						'default' => '',
 					),
-					'classname'  => array(
+					'classname'    => array(
 						'type'    => 'string',
 						'default' => '',
 					),
-					'rel'        => array(
+					'rel'          => array(
 						'type'    => 'string',
 						'default' => '',
 					),
 					'display_link' => array(
 						'type' => 'boolean',
 						'default' => true,
 					),
-					'image_size' => array(
+					'image_size'   => array(
 						'type'    => 'string',
 						'default' => 'medium',
 					),
-					'content'    => array(
+					'content'      => array(
 						'type'    => 'string',
 						'default' => '',
 					),
+					'display_link' => array(
```

Comment:
The attribute is listed already on line 159. The only difference is referring to the type as boolean rather than bool. I couldn't find any docs to say which is correct?

Reply:
I think I removed this one when I found it.

Reply:
oh - it was removed in the next commit. Apologies :)